packages/framework/src/Model/Payment/Service/PaymentServiceFacade.php

```diff
@@ -102,7 +103,7 @@ public function updatePaymentTransactionsByOrder(Order $order): void
                 if ($update) {
                     $this->paymentTransactionFacade->edit($paymentTransaction->getId(), $paymentTransactionData);
                 }
-            } catch (PaymentServiceFacadeNotRegisteredException|GoPayNotConfiguredException|PaymentTransactionHasNoAssignedPayment $exception) {
+            } catch (PaymentServiceFacadeNotRegisteredException|GoPayNotConfiguredException|GoPayNotEnabledOnDomainException|PaymentTransactionHasNoAssignedPayment $exception) {
                 $this->logger->error($exception->getMessage());
```

Comment:
:bulb: 

It is not your code. I've noticed logging error without setting context. From my point of view, it is good practice to pass exception into context, it will pass stack trace to Sentry on exception

Source: https://github.com/getsentry/sentry-php/blob/master/src/Monolog/Handler.php#L63

Reply:
nice, thanks :)  added in 88eae2e5fc